plugins/by-name/lsp-signature/default.nix

```diff
@@ -0,0 +1,212 @@
+{ lib, helpers, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "lsp-signature";
+  originalName = "lsp_signature.nvim";
+  package = "lsp_signature-nvim";
+  luaName = "lsp_signature";
+
+  maintainers = [ lib.maintainers.wadsaek ];
+
+  settingsOptions = {
+    debug = defaultNullOpts.mkBool false ''
+      Whether to enable debug logging.
+    '';
+
+    log_path =
+      defaultNullOpts.mkStr
+        (lib.nixvim.mkRaw # lua
+          ''
+            vim.fn.stdpath("cache") .. "/lsp_signature.log"
+          ''
+        )
+        ''
+          Log directory for when debug is on.
+          Default is `~/.cache/nvim/lsp_signature.log`.
```

Comment:
We don't normally need to state the default in the description text, because the docs will render the default and the description right next to each other :slightly_smiling_face:

Reply:
the plugin default has some lua shenanigans, I thought it'd make sense to keep the comment from the original docs